README.md

```diff
@@ -0,0 +1,9 @@
+# generator-testcafe-browser-provider
+*Scaffold out a TestCafe browser provider*
+
+[![Build Status](https://travis-ci.org/DevExpress/generator-testcafe-browser-provider.svg)](https://travis-ci.org/DevExpress/generator-testcafe-browser-provider)
+
+If you want to setup tests with your in-company server farm, or your browsers are started in some unusual way, TestCafe can handle it. What you have to do is create a custom browser provider plugin.
+This Yeoman generator scaffolds out such a plugin, so that you only need to write a few lines of code.
+
+For more information, read _Custom Browser Provider Plugin link here_
```

Comment:
I'll also change the report generator readme according to some corrections here.